docs/specifications/4-standard-compliance-suite.md

```diff
@@ -0,0 +1,166 @@
+# Standard Compliance Suite
+
+## Format
+In order to assert whether a new runtime implementation is correct, a formal
+compliance test suite is defined, that all new language runtimes must fully
+implement before they can be deemed eligible for General Availability.
+
+This document describes these tests, as well as a general approach for ensuring
+conformance requirements are met in a systematic manner.
+
+### Categories
```

Comment:
I am not sure I understand what categories are? Who defines them? Why do we need them?

Reply:
I actually mean to define a set of standard categories that tests need to be in. Refrained from trying to create that list upfront though, so I also didn't mention that would exist... But it was my idea (until it is proven bad by experience?)